versionsOfDependencies.gradle

```diff
@@ -59,4 +59,5 @@ ext {
     revSlf4j = '1.7.25'
     revSlf4jlog4j = '1.8.0-alpha1'
     revSpectator = '0.68.0'
+    revAmqpClient = '5.6.0'
```

Comment:
Please keep this file in alphabetical order.

Reply:
I've just pushed to modifications